samples/compute_enable_oslogin_project.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: constraints.gatekeeper.sh/v1alpha1
+kind: GCPComputeEnableOSLoginProjectConstraintV1
+metadata:
+  name: compute-enable-oslogin-project
+  annotations:
+    bundles.validator.forsetisecurity.org/cis-v1.1: 4.04
+    description: Checks if "oslogin" is enabled for a Project
```

Comment:
Is this an accurate description? From the Rego, it looks like this actually checks that OS login is enabled on each VM in the proejct.